src/test/java/io/xstefank/wildfly/bot/utils/Util.java

```diff
@@ -0,0 +1,43 @@
+package io.xstefank.wildfly.bot.utils;
+
+import io.quarkiverse.githubapp.testing.GitHubAppMockito;
+import io.quarkiverse.githubapp.testing.dsl.GitHubMockSetupContext;
+import io.xstefank.wildfly.bot.model.RuntimeConstants;
+import org.kohsuke.github.GHCommitState;
+import org.kohsuke.github.GHPullRequestCommitDetail;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.PagedSearchIterable;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.processEmptyPullRequestMock;
+
+public class Util {
+
+    public static void prepareMockedRepo(GitHubMockSetupContext mocks, String wildflyConfigFile, GitHubJson gitHubJson, String commitMessage) throws IOException {
+        mocks.configFile(RuntimeConstants.CONFIG_FILE_NAME).fromString(wildflyConfigFile);
+        GHPullRequestCommitDetail mockCommitDetail = Mockito.mock(GHPullRequestCommitDetail.class);
+        Mockito.when(mockCommitDetail.getSha()).thenReturn(gitHubJson.commitSHA());
+        PagedSearchIterable<GHPullRequestCommitDetail> commitDetails = GitHubAppMockito.mockPagedIterable(mockCommitDetail);
+        Mockito.when(mocks.pullRequest(gitHubJson.id()).listCommits()).thenReturn(commitDetails);
+
+        if (commitMessage != null) {
+            GHPullRequestCommitDetail.Commit mockCommit = Mockito.mock(GHPullRequestCommitDetail.Commit.class);
+            Mockito.when(mockCommitDetail.getCommit()).thenReturn(mockCommit);
+            Mockito.when(mockCommit.getMessage()).thenReturn(commitMessage);
+        }
+
+        processEmptyPullRequestMock(mocks.pullRequest(gitHubJson.id()));
```

Comment:
I have started to work on builder for mocking different github objects. Perhaps that might be better approach rather than pass single `commitMessage`. We might need to mock more messages or perhaps new objects in the future. See here https://github.com/The-Huginn/wildfly-github-bot/blob/notify-to-pr-review/src/test/java/io/xstefank/wildfly/bot/helper/MockedGHPullRequestProcessor.java